### PR TITLE
Fixes #28687: Drop setting installer dir for Katello on build

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Review by evgeni — APPROVED** [review]

File: Rakefile [hunk 1]
```diff
@@ -106,11 +106,6 @@ SCENARIOS.each do |scenario|
       'parser_cache_path' => "#{DATADIR}/foreman-installer/parser_cache/#{scenario}.yaml",
     }
 
-    # These scenarios have their own hook dirs for now
-    if ['foreman-proxy-content', 'katello'].include?(scenario)
-      scenario_config_replacements['installer_dir'] += '/katello'
```
Comment (ekohl) [inline Rakefile:111]: This should now be replacing the hook dirs. Since it's an array, sed might be harder to use.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: Rakefile (hunk 1 above)
Comment (ehelms) [reply Rakefile:111]: Technically, the migrations handle this for us in the end so having it in this build step is not needed. I'm trying to combine the hooks to a single directory anyway which would then remove any hook_dir updating code I added.

**Review by ekohl — APPROVED** [review]
In itself this is a correct change. Not sure I'm too happy about the hooks dir and it depending on migrations to deploy. Somehow I expect it to come around and bite us.

**Comment by ehelms** [comment]
I'll keep an eye out for any oddities and keep driving the combination of hooks to possibly avoid this all together.

**Comment by ekohl** [comment]
From the katello upgrade pipeline it looks like the hook dir isn't correctly set:
```
TASK [foreman_installer : Run installer] ***************************************
fatal: [pipeline-upgrade-katello-nightly-centos7]: FAILED! => changed=true 
  cmd: |-
    foreman-installer -v --no-colors --disable-system-checks --scenario katello  --upgrade --certs-update-all --upgrade --certs-update-all --foreman-initial-admin-password changeme --puppet-server-max-active-instances 1 --puppet-server-jvm-min-heap-size 1G --puppet-server-jvm-max-heap-size 1G
  delta: '0:00:05.970213'
  end: '2020-01-11 14:14:30.748426'
  msg: non-zero return code
  rc: 1
  start: '2020-01-11 14:14:24.778213'
  stderr: |-
    tput: unknown terminal "unknown"
    ERROR: Unrecognised option '--disable-system-checks'
  
    See: 'foreman-installer --help'
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>
```

**Comment by ekohl** [comment]
From foreman-installer-2.0.0-0.1.develop.20200110141718git618f175.el7.noarch in `/etc/foreman-installer/scenarios.d/katello.yaml`:
```yaml
---
:name: Katello
:description: Install Foreman with Katello
:enabled: true
:log_dir: /var/log/foreman-installer
:store_dir: ''
:log_name: katello.log
:log_level: DEBUG
:no_prefix: false
:mapping: {}
:answer_file: /etc/foreman-installer/scenarios.d/katello-answers.yaml
:installer_dir: /usr/share/foreman-installer
:module_dirs: /usr/share/foreman-installer/modules
:colors: true
:color_of_background: :dark
:hook_dirs:
- ./katello/hooks
:custom: {}
:facts:
  tuning: default
:low_priority_modules: []
:verbose_log_level: info
:skip_puppet_version_check: false
:parser_cache_path: /usr/share/foreman-installer/parser_cache/katello.yaml
:hiera_config: /usr/share/foreman-installer/config/foreman-hiera.yaml
:order:
- certs
- foreman
- katello
- foreman_proxy
- foreman_proxy::plugin::pulp
- foreman_proxy_content
- puppet
```

So as I expected in https://github.com/theforeman/foreman-installer/pull/441#discussion_r364242651, the migrations didn't take care of this on fresh installations.

**Pull request merged**